docs/README.md

```diff
@@ -6,11 +6,9 @@ The documentation for Compose has been merged into
 The docs for Compose are now here:
 https://github.com/docker/docker.github.io/tree/master/compose
 
-Please submit pull requests for unpublished features on the `vnext-compose` branch (https://github.com/docker/docker.github.io/tree/vnext-compose).
+Please submit pull requests for unpublished on the `master` branch (https://github.com/docker/docker.github.io/tree/master).
```

Comment:
Could you please add instructions on marking a PR as WIP in the title for PRs on unreleased features. For enhancements to existing content, your statement above applies.

Reply:
done, thanks!